charts/civil-ccd/values.preview.template.yaml

```diff
@@ -228,13 +229,6 @@ ccd:
         IDAM_USER_URL: https://idam-web-public.aat.platform.hmcts.net
       keyVaults: []
       ingressHost: ccd-definition-store-${SERVICE_FQDN}
-  ccd-user-profile-api:
```

Comment:
Dont we need to remove it in values.aat.template yaml?

Reply:
Done